tests/operations/docker.container/add_existent_container.json

```diff
@@ -9,242 +9,240 @@
-                                },
-                                "NetworkSettings": {
-                                    "Bridge": "",
-                                    "SandboxID": "30102172778c8c2268fa443d0c29ac898beaa07c6ca3d73a93751a21e0812f14",
-                                    "SandboxKey": "/var/run/docker/netns/30102172778c",
-                                    "Ports": {
-                                        "80/tcp": [
-                                            {
-                                                "HostIp": "0.0.0.0",
-                                                "HostPort": "80"
-                                            }
-                                        ]
-                                    },
-                                    "HairpinMode": "False",
-                                    "LinkLocalIPv6Address": "",
-                                    "LinkLocalIPv6PrefixLen": 0,
-                                    "SecondaryIPAddresses": "None",
-                                    "SecondaryIPv6Addresses": "None",
-                                    "EndpointID": "2e0e98fd4346d9ad61d78294bd97e7c27f512a88c402115163a0df73bf83cad4",
-                                    "Gateway": "172.17.0.1",
-                                    "GlobalIPv6Address": "",
-                                    "GlobalIPv6PrefixLen": 0,
-                                    "IPAddress": "172.17.0.2",
-                                    "IPPrefixLen": 16,
-                                    "IPv6Gateway": "",
-                                    "MacAddress": "02: 42:ac: 11: 00: 02",
-                                    "Networks": {
-                                        "bridge": {
-                                            "IPAMConfig": "None",
-                                            "Links": "None",
-                                            "Aliases": "None",
-                                            "MacAddress": "02: 42:ac: 11: 00: 02",
-                                            "NetworkID": "9602c89bf5d2a3b55665598ec99ec803cd54e0df0c2a25a511d59ecc2dc047b5",
-                                            "EndpointID": "2e0e98fd4346d9ad61d78294bd97e7c27f512a88c402115163a0df73bf83cad4",
-                                            "Gateway": "172.17.0.1",
-                                            "IPAddress": "172.17.0.2",
-                                            "IPPrefixLen": 16,
-                                            "IPv6Gateway": "",
-                                            "GlobalIPv6Address": "",
-                                            "GlobalIPv6PrefixLen": 0,
-                                            "DriverOpts": "None",
-                                            "DNSNames": "None"
-                                        }
-                                    }
+        "docker.DockerContainer": {
```

Comment:
Apologies for the diff here; the diffs in this fact object should be purely whitespace